index.js

```diff
@@ -16,10 +18,6 @@
 
 const { setUnderTest } = require('./lib/utils/utils');
 setUnderTest(); // Note: we must call setUnderTest before initializing.
-
-const { Playwright } = require('./lib/server/playwright');
-const { setupInProcess } = require('./lib/inprocess');
-const path = require('path');
-
-const playwright = new Playwright(__dirname, require(path.join(__dirname, 'browsers.json'))['browsers']);
-module.exports = setupInProcess(playwright);
+module.exports = require('./lib/inprocess');
+if (require.main === module)
```

Comment:
this is a strange python pattern. The standard way ive seen is to set the "bin" field in `package.json` with a separate file. See https://github.com/microsoft/folio/blob/master/package.json#L4

Reply:
Done.